kfs-core/src/main/resources/edu/arizona/kfs/fp/businessobject/datadictionary/PaymentMethod.xml

```diff
@@ -26,6 +26,14 @@
 
 <!-- Attribute Definitions -->
 
+	<!--
+		New singleton for the payment method values finder, can be referenced by all uses.
+	-->
+	<bean name="PaymentMethodSelectControl" parent="SelectControlDefinition" p:valuesFinderClass="edu.arizona.kfs.sys.businessobject.options.PaymentMethodValuesFinder" />
+	<bean name="PaymentMethodsForDVSelectControl" parent="SelectControlDefinition" p:valuesFinderClass="edu.arizona.kfs.fp.businessobject.options.PaymentMethodsForDVValuesFinder" />
```

Comment:
The PaymentMethodsForDVSelectControl (see DisbursementVoucherDocument DD lines 22-24) and PaymentMethodsForVendorSelectControl (see VendorDetailExtension BO DD lines 48-50) beans are not needed. Please remove.